src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/arm/pooling.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "custom/single_layer_tests/classes/pooling.hpp"
+#include "utils/cpu_test_utils.hpp"
+#include "utils/fusing_test_utils.hpp"
+#include "utils/filter_cpu_info.hpp"
+
+using namespace CPUTestUtils;
+
+namespace ov {
+namespace test {
+namespace Pooling {
+
+const std::vector<maxPoolV8SpecificParams>& paramsMaxV144D_2x2kernel = {
+        maxPoolV8SpecificParams{ {2, 2}, {2, 2}, {1, 1}, {0, 0}, {0, 0},
+                                                          ov::element::Type_t::i32, 0,
+                                                          ov::op::RoundingType::CEIL_TORCH, ov::op::PadType::SAME_UPPER },
+        maxPoolV8SpecificParams{ {2, 2}, {2, 2}, {1, 1}, {0, 0}, {0, 0},
+                                                          ov::element::Type_t::i32, 0,
+                                                          ov::op::RoundingType::CEIL_TORCH, ov::op::PadType::SAME_LOWER }
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_MaxPoolV14_CPU_4D_2x2Kernel, MaxPoolingV14LayerCPUTest,
```

Comment:
I think we already have common Pooling test with such params. Should we have another one?

Reply:
There is no MaxPoolV14 common test.
MaxPoolV14 tests declared in common file are wrapped by `#if defined(OPENVINO_ARCH_X86) || defined(OPENVINO_ARCH_X86_64)`